models/parkingLot.js

```diff
@@ -0,0 +1,76 @@
+const Slot = require("./slot");
+
+class ParkingLot {
+  constructor(fileData, lotSize = 10) {
```

Comment:
Why is the default value for `lotSize` mentioned here directly as 10? Wasn't it moved to a constant?